src/rpc/blockchain.cpp

```diff
@@ -331,18 +331,18 @@ UniValue getdifficulty(const UniValue& params, bool fHelp)
 
 std::string EntryDescriptionString()
 {
-    return "    \"size\" : n,             (numeric) transaction size in bytes\n"
+    return "    \"size\" : n,             (numeric) virtual transaction size. Differs from size for witness transactions since witness size is discounted. BIP 141.\n"
```

Comment:
Seems reasonable, though you might have meant "See BIP 141" instead of just "BIP 141" :).